app/src/main/AndroidManifest.xml

```diff
@@ -72,7 +72,6 @@
         <activity
             android:name=".recipe.RecipeDetailsActivity"
             android:exported="true" />
-        <activity android:name=".recipe.RecipeStorageActivity" />
```

Comment:
Why is the storage activity removed from here?

Reply:
Fixed in latest commit